src/main/java/com/openlattice/Conductor.java

```diff
@@ -69,7 +66,7 @@
     }
 
     public Conductor() {
-        super( Pods.concatenate( RhizomeApplicationServer.DEFAULT_PODS, rhizomePods, conductorPods ) );
```

Comment:
Need to keep `RhizomeApplicationServer.DEFAULT_PODS` as that has the `RegistryBasedHazelcastInstanceConfigurationPod` and a couple of others

Reply:
I know, but it will get added in the `RhizomeApplicationServer` constructors anyways, so no need to double add it.
```   
 public RhizomeApplicationServer( Class<?>... pods ) {
        this( DEFAULT_PODS, pods );
    }
```

Reply:
Oh awesome lol, totally didn't see that.

Okay so then you also don't need `Pods.concatenate()`, you can just pass `conductorPods` directly

Reply:
Yeah, right! will do